katz centrality...
Traceback (most recent call last):
  File "/home/vassm/KGEntityMatching/centrality_vector.py", line 63, in <module>
    centrality_dict = nx.katz_centrality(G)
  File "/home/vassm/.conda/envs/python3_10/lib/python3.10/site-packages/networkx/utils/decorators.py", line 788, in func
    return argmap._lazy_compile(__wrapper)(*args, **kwargs)
  File "<class 'networkx.utils.decorators.argmap'> compilation 17", line 4, in argmap_katz_centrality_13
  File "/home/vassm/.conda/envs/python3_10/lib/python3.10/site-packages/networkx/utils/backends.py", line 967, in __call__
    return self.orig_func(*args, **kwargs)
  File "/home/vassm/.conda/envs/python3_10/lib/python3.10/site-packages/networkx/algorithms/centrality/katz.py", line 194, in katz_centrality
    raise nx.PowerIterationFailedConvergence(max_iter)
networkx.exception.PowerIterationFailedConvergence: (PowerIterationFailedConvergence(...), 'power iteration failed to converge within 1000 iterations')





current flow betweenness centrality...
Traceback (most recent call last):
  File "/home/vassm/KGEntityMatching/centrality_vector.py", line 77, in <module>
    centrality_dict = nx.current_flow_betweenness_centrality(G)
  File "/home/vassm/.conda/envs/python3_10/lib/python3.10/site-packages/networkx/utils/decorators.py", line 788, in func
    return argmap._lazy_compile(__wrapper)(*args, **kwargs)
  File "<class 'networkx.utils.decorators.argmap'> compilation 18", line 4, in argmap_current_flow_betweenness_centrality_14
  File "/home/vassm/.conda/envs/python3_10/lib/python3.10/site-packages/networkx/utils/backends.py", line 967, in __call__
    return self.orig_func(*args, **kwargs)
  File "/home/vassm/.conda/envs/python3_10/lib/python3.10/site-packages/networkx/algorithms/centrality/current_flow_betweenness.py", line 223, in current_flow_betweenness_centrality
    raise nx.NetworkXError("Graph not connected.")
networkx.exception.NetworkXError: Graph not connected.





communicability betweenness centrality...
/home/vassm/.conda/envs/python3_10/lib/python3.10/site-packages/networkx/algorithms/centrality/subgraph_alg.py:277: RuntimeWarning: invalid value encountered in divide
  B = (expA - sp.linalg.expm(A)) / expA





information centrality...
Traceback (most recent call last):
  File "/home/vassm/KGEntityMatching/centrality_vector.py", line 79, in <module>
    centrality_dict = nx.information_centrality(G)
  File "/home/vassm/.conda/envs/python3_10/lib/python3.10/site-packages/networkx/utils/decorators.py", line 788, in func
    return argmap._lazy_compile(__wrapper)(*args, **kwargs)
  File "<class 'networkx.utils.decorators.argmap'> compilation 5", line 4, in argmap_current_flow_closeness_centrality_1
  File "/home/vassm/.conda/envs/python3_10/lib/python3.10/site-packages/networkx/utils/backends.py", line 967, in __call__
    return self.orig_func(*args, **kwargs)
  File "/home/vassm/.conda/envs/python3_10/lib/python3.10/site-packages/networkx/algorithms/centrality/current_flow_closeness.py", line 72, in current_flow_closeness_centrality
    raise nx.NetworkXError("Graph not connected.")
networkx.exception.NetworkXError: Graph not connected.





second order centrality...
Traceback (most recent call last):
  File "/home/vassm/KGEntityMatching/centrality_vector.py", line 85, in <module>
    centrality_dict = nx.second_order_centrality(G)
  File "/home/vassm/.conda/envs/python3_10/lib/python3.10/site-packages/networkx/utils/decorators.py", line 788, in func
    return argmap._lazy_compile(__wrapper)(*args, **kwargs)
  File "<class 'networkx.utils.decorators.argmap'> compilation 15", line 4, in argmap_second_order_centrality_11
  File "/home/vassm/.conda/envs/python3_10/lib/python3.10/site-packages/networkx/utils/backends.py", line 967, in __call__
    return self.orig_func(*args, **kwargs)
  File "/home/vassm/.conda/envs/python3_10/lib/python3.10/site-packages/networkx/algorithms/centrality/second_order.py", line 109, in second_order_centrality
    raise nx.NetworkXException("Non connected graph.")
networkx.exception.NetworkXException: Non connected graph.
